evolver/device.py

```diff
@@ -25,6 +25,9 @@ class Config(BaseInterface.Config):
         history: ConfigDescriptor | History = ConfigDescriptor.model_validate(DEFAULT_HISTORY)
         enable_control: bool = True
         interval: int = settings.DEFAULT_LOOP_INTERVAL
+        raise_loop_exceptions: bool = False
+        abort_on_control_errors: bool = False
+        skip_control_on_read_failure: bool = True
```

Comment:
I don't think this gives us what you were looking for. Looks like it's missing ``abort_on_commit _errors`` or similarly named.

As we have it here, to raise commit errors ``raise_loop_exceptions`` needs to be ``True``. However, this is only checked in ``_loop_exception_wrapper`` and therefore takes precedence over both ``abort_on_control_errors`` & ``skip_control_on_read_failure``. This means the scenario where you don't raise on read and control errors but do on commit isn't covered, right?

I would have thought raising on commit errors to be the most important - assuming we make any distinction at all.

Reply:
😮